packages/python/pyproject.toml

```
@@ -4,7 +4,7 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "lsprotocol"
-description = 'Python implementation of the Language Server Protocol.'
+description = 'Python-based Language Server Protocol types code generator for various languages.'
```

```suggestion
description = 'Python types for Language Server Protocol.'
```
This toml is only for the types. It is not for the generator.